### DISCUSSION TITLE
[TOOL] autorestart server tool working for AZ?

### DISCUSSION THREAD

**Discussion by chamorrorenzo** [azerothcore/azerothcore-wotlk#3897] [Tools]
Im trying to use the one that I used to with trinity but doesnt seem to work on AZ
is there any tool to use instead?

thanks

**Comment by FrancescoBorzi** [comment]
https://stackoverflow.com/questions/62910364/azerothcore-with-gdb-restarter-and-crashlog

**Comment by nolt** [comment]
Maybe you looking for restarter, you can give a hit one that I use:
https://github.com/nolt/acore-linux-restarter

**Comment by Yehonal** [comment]
Another way is to use `./acore.sh run-worlserver` and `./acore.sh run-authserver` 

If you're on linux you can also use tmux + systemd to start them at system startup
